front/src/views/Signin.vue

```diff
@@ -0,0 +1,49 @@
+<template>
```

Comment:
Esta parte no se como revisarla porque es in Signin pero por la cara tiene in axios que ataca al get de todos los eventos y no lo entiendo